scene/3d/gpu_particles_collision_3d.cpp

```diff
@@ -866,7 +943,13 @@ void GPUParticlesAttractor3D::_bind_methods() {
 	ADD_PROPERTY(PropertyInfo(Variant::FLOAT, "strength", PROPERTY_HINT_RANGE, "-128,128,0.01,or_greater,or_less"), "set_strength", "get_strength");
 	ADD_PROPERTY(PropertyInfo(Variant::FLOAT, "attenuation", PROPERTY_HINT_EXP_EASING, "0,8,0.01"), "set_attenuation", "get_attenuation");
 	ADD_PROPERTY(PropertyInfo(Variant::FLOAT, "directionality", PROPERTY_HINT_RANGE, "0,1,0.01"), "set_directionality", "get_directionality");
-	ADD_PROPERTY(PropertyInfo(Variant::INT, "cull_mask", PROPERTY_HINT_LAYERS_3D_RENDER), "set_cull_mask", "get_cull_mask");
+	ADD_PROPERTY(PropertyInfo(Variant::INT, "collision_layer", PROPERTY_HINT_LAYERS_3D_PARTICLE_COLLISION), "set_collision_layer", "get_collision_layer");
+
+#ifndef DISABLE_DEPRECATED
+
+	ADD_PROPERTY(PropertyInfo(Variant::INT, "cull_mask", PROPERTY_HINT_LAYERS_3D_PARTICLE_COLLISION), "set_cull_mask", "get_cull_mask");
+
+#endif //DISABLE_DEPRECATED
```

Comment:
```suggestion
#ifndef DISABLE_DEPRECATED
	ADD_PROPERTY(PropertyInfo(Variant::INT, "cull_mask", PROPERTY_HINT_LAYERS_3D_PARTICLE_COLLISION, "", PROPERTY_USAGE_NONE), "set_cull_mask", "get_cull_mask");
#endif // DISABLE_DEPRECATED
```
This should make sure they are neither stored nor visible in the editor